src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml

```diff
@@ -0,0 +1,424 @@
+security:
+    login:
+        page_title: 'Benvenguda sus wallabag !'
+        keep_logged_in: 'Demorar connectat'
+        forgot_password: 'Senhal doblidat ?'
+        submit: 'Se connectar'
+        register: 'Crear un compte'
+        username: "Nom d'utilizaire"
+        password: 'Senhal'
+        cancel: 'Anullar'
+    resetting:
+        description: "Picatz vòstra adreça de corrièl çai-jos, vos mandarem las instruccions per reïnicializar vòstre senhal."
+    register:
+        page_title: 'Se crear un compte'
+        go_to_account: 'Anar sus vòstre compte'
+
+menu:
+    left:
+        unread: 'Pas legits'
+        starred: 'Favorits'
+        archive: 'Legits'
+        all_articles: 'Tots los articles'
+        config: 'Configuracion'
+        tags: 'Etiquetas'
+        internal_settings: 'Configuracion interna'
+        import: 'Importar'
+        howto: 'Ajuda'
+        developer: 'Desvolopador'
+        logout: 'Déconnexion'
+        about: 'A prepaus'
+        search: 'Cercar'
+        save_link: 'Enregistrar un novèl article'
+        back_to_unread: 'Tornar als articles pas legits'
+    top:
+        add_new_entry: 'Enregistrar un novèl article'
+        search: 'Cercar'
+        filter_entries: 'Filtrar los articles'
+        export: 'Exportar'
+    search_form:
+        input_label: 'Picatz vòstre mot-clau a cercar aquí'
+
+footer:
+    wallabag:
+        elsewhere: 'Emportatz wallabag amb vosaultres'
+        social: 'Social'
+        powered_by: 'propulsat per'
+        about: 'A prepaus'
+    page_title: 'Configuracion'
+
+config:
+    tab_menu:
+        settings: 'Paramètres'
+        rss: 'RSS'
+        user_info: 'Mon compte'
+        password: 'Senhal'
+        rules: 'Règlas d\'etiquetas automaticas'
```

Comment:
Can you remove the `d\'` part, and put the string into double quotes : `"Règlas d'etiquetas automaticas"`